### PR TITLE
ng-href

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: app/directives/bbi/menu.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ define(['app', 'text!./menu.html','lodash','providers/locale'], function(app, te
             link: function ($scope) {
               $scope.locale=locale;
 
-                  require(["_slaask"], function(_slaask) {
+                  require(["js/slaask"], function(_slaask) {
```
Comment (sbilodeau) [inline app/directives/bbi/menu.js]: do you have a different template?

Comment (sbilodeau) [reply app/directives/bbi/menu.js]: https://github.com/scbd/www.cbd.int/blob/master/app/template.js#L41

File: app/directives/bbi/slider.html [hunk 2]
```diff
@@ -101,9 +101,9 @@ <h5 class="main-title"><img src="/app/images/bbi/logo_white.svg" style="height:7
         <div class="carousel-inner" role="listbox" style="height:400px;">
 
             <div class="item pointer" ng-repeat="item in carouselData.items" ng-class="{active: $index==0}">
-                <div class="img-cont" ng-click="go(item.targetUrl)" >
+                <a class="img-cont" style="display:block;" ng-href="{{item.targetUrl}}" ng-target="{{item.target||'_self'}}">
```
Comment (sbilodeau) [inline app/directives/bbi/slider.html]: dis you test `display:block` on IE?

File: app/directives/bbi/menu.js (hunk 1 above)
Comment (randyhoulahan) [reply app/directives/bbi/menu.js]: different routing

**Comment by randyhoulahan** [comment]
works

File: app/directives/bbi/slider.html (hunk 2 above)
Comment (randyhoulahan) [reply app/directives/bbi/slider.html]: :+1:

**Pull request merged**